qcfractal/interface/models/records.py

```diff
@@ -90,6 +90,10 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return f"<{self}>"
 
+    def __eq__(self, other) -> bool:
```

Comment:
Lets remove all of these and let LGTM complain.

Reply:
Agreed. I'll strip all those out.